src/com/amazon/corretto/crypto/provider/EvpSignature.java

```diff
@@ -324,17 +328,29 @@ private EvpSignature(AmazonCorrettoCryptoProvider provider, final EvpKeyType key
         Loader.checkNativeLibraryAvailability();
         digestName_ = digestName;
 
-        signingBuffer = new InputBuffer<byte[], EvpContext, RuntimeException>(1024)
+        signingBuffer = getSigningBuffer();
+        verifyingBuffer = getVerifyingBuffer();
+    }
+
+    private InputBuffer<byte[], EvpContext, RuntimeException> getSigningBuffer() {
+        final String pssMgfMd;
+        if (pssParams != null) {
```

Comment:
FYI, this is the line which is failing the CI right now.
```suggestion
        if (pssParams_ != null) {
```